### PR TITLE
Release 0.13.0

### PR DESCRIPTION
**Pull request by frostming**
Signed-off-by: Frost Ming <me@frostming.com>

0.13.0
======

* Discard the ``strict`` attribute when serializing and deserializing responses.
* Fix the ``IncompleteRead`` error thrown by ``urllib3 2.0``.
* Remove usage of ``utcnow`` in favor of timezone-aware datetimes.
* Remove the ``compat`` module.
* Use Python's ``unittest.mock`` library instead of ``mock``.
* Add type annotations.
* Exclude the ``tests`` directory from the wheel.

Any other to include? @woodruffw

### CODE REVIEW

**Comment by woodruffw** [comment]
> Signed-off-by: Frost Ming <me@frostming.com>
> 
> Any other to include? @woodruffw

Nope, LGTM! I'd say we could also include a note in the changelog about the change of repo URL/org, but I'll defer to your judgment 🙂

(I have some other changes I want to queue up, but they can wait.)

**Review by woodruffw — APPROVED** [review]

**Comment by frostming** [comment]
> Nope, LGTM! I'd say we could also include a note in the changelog about the change of repo URL/org, but I'll defer to your judgment 🙂

Done, but I think the [docs site](https://cachecontrol.readthedocs.io/en/latest/release_notes.html) has not been updated for a while. @ionrock can you grant me access as well?

**Comment by ionrock** [comment]
@frostming I added you as a maintainer. Thanks!

**Pull request merged**
